coq.opam

```diff
@@ -27,7 +27,7 @@ depends: [
   "num"
 ]
 
-build: [
-  [ "./configure" "-prefix" prefix ]
-  [ "dune" "build" "-p" name "-j" jobs ]
+build-env: [
+  [ COQ_CONFIGURE_PREFIX = "%{prefix}" ]
```

Comment:
In view of the discussion and #11898 , I suggest we just revert #11497

Reply:
Isn't that going to disable precompilation of native files for bignums users?

Reply:
As we don't compile yet the stdlib with native we have no other choice given the new rules, hopefully that will be fixed soon along with https://github.com/coq/coq-bench/issues/74

Reply:
But this is only for Dune-compiled Coq, right?

Reply:
Yes, #8860 is still not addressed, but almost there.

Reply:
Why do we use dune in the .opam considering it forces disabling features? What is the goal of the .opam?

Reply:
@SkySkimmer the history is complex, we added and opam file as Dune used to require them for identifying packages, at first it was empty, then we made it similar to the auto-generated opam files; it is still pending for us to bump to actually use that Dune auto-generation facility. At the time there was no problem with this setup as no other development was actually using `-native=yes`; it was also the case the non-dune opam files where quite buggy due to the switch to 2.0, having problems with the sandbox list of registered files so the dune setup was more stable, so we keep this for the bench which has been so far rock-solid. OPAM 2.0 also brought an impacting change, originally, the opam file in the repos wasn't picked up by opam 1.2, but that changed with 2.0 so then people doing pinning were using the private Coq opam file which was certainly not intended.